docs-vim.js

```diff
@@ -1,3 +1,4 @@
+num = '';
```

Comment:
Can you move `num` to be under the `vim` object? Just to keep everything relatively well-scoped.